.github/workflows/edge-test.yml

```
@@ -0,0 +1,101 @@
+        working-directory: lib/edge
+        run: just rs-examples
+
+      - name: 🐍 Build Python edge
+        working-directory: lib/edge/python
+        run: just py-build
+
+      - name: 🐍 Run Python examples
+        working-directory: lib/edge
+        run: just py-examples
+
+      - name: Check for uncommitted changes
```

actionlint (1.7.11)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)